arctic/arctic.py

```diff
@@ -202,6 +213,44 @@ def list_libraries(self):
                         libs.append(coll[:-1 * len(self.METADATA_COLL) - 1])
         return libs
 
+    @mongo_retry
+    def invalidate_cache(self):
+        """
+        Deletes the data in the collection used to cache the result
+        of list_libraries
+        :return:
+        """
+        self._conn.meta_db.cache.drop()
```

Comment:
Not sure about drop, here - it may be reasonably heavyweight in the database. Can we just delete_many or similar?